packages/@react-spectrum/tabs/test/Tabs.test.js

```diff
@@ -24,13 +24,13 @@ let items = [
   {name: 'Tab 3', children: 'Tab 3 body'}
 ];
 
-function renderComponent(props) {
+function renderComponent(props, itemProps) {
   return render(
     <Provider theme={theme}>
       <Tabs {...props} items={items}>
         <TabList>
           {item => (
-            <Item key={item.name} title={item.name || item.children} />
+            <Item key={item.name} title={item.name || item.children} {...itemProps} />
```

Comment:
I think we'll want to spread first, then set the critical props
```suggestion
            <Item {...itemProps} key={item.name} title={item.name || item.children} />
```

Reply:
Otherwise the critical props could be overwritten by the other props. Good catch, thanks! Resolved.